packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx

```diff
@@ -182,6 +182,20 @@ const getAlertContent = (message, data = undefined) => {
           defaultMessage='Currency has been successfully updated'
         />
       )
+    case C.DEBIT_CARD_DASHBOARD_LOCK_ERROR:
+      return buildMessageTemplate(
+        <FormattedMessage
+          id='components.alerts.debit_card_dashboard_lock_error'
+          defaultMessage='Failed to change Card Lock'
```

Comment:
does "Card Lock" need to be capitalized?

Reply:
Discussions with the UX about these messages are still pending